docs/source/converting.rst

```diff
@@ -34,7 +34,7 @@ Move the model architecture and forward pass to your :class:`~pytorch_lightning.
 
 2. Move the optimizer(s) and schedulers
 =======================================
-Move your optimizers to :func:`pytorch_lightning.core.LightningModule.configure_optimizers` hook. Make sure to use the hook parameters (self in this case).
+Move your optimizers to :func:`~pytorch_lightning.core.LightningModule.configure_optimizers` hook. Make sure to use the hook parameters (self in this case).
```

Comment:
What does this comment even mean?
"Make sure to use the hook parameters (self in this case)."

Reply:
May be @edenlightning can explain it. I just edited the `~`

Reply:
I think that this is not needed...

> Make sure to use the hook parameters (self in this case).

Reply:
It's been removed.